src/components/BrowseResultsPane/BrowseResultsPane.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Pane,
+  PaneMenu,
+} from '@folio/stripes/components';
+import {
+  AppIcon,
+  useNamespace,
+} from '@folio/stripes/core';
+import {
+  ExpandFilterPaneButton,
+} from '@folio/stripes-smart-components';
+import {
+  getFiltersCount,
+} from '@folio/stripes-acq-components';
+
+const BrowseResultsPane = ({
+  filters = {},
+  isFiltersOpened,
+  toggleFiltersPane,
+}) => {
+  const [namespace] = useNamespace();
+
+  const paneSub = <FormattedMessage id="ui-inventory.title.subTitle.browseCall" />;
+
+  const firstMenu = useMemo(() => (
+    isFiltersOpened
+      ? null
+      : (
+        <PaneMenu>
+          <ExpandFilterPaneButton
+            filterCount={getFiltersCount(filters)}
```

Comment:
please chech if `filters` var should be included in dependency array

Reply:
Good catch, thanks!